subgraphs/collectible-market/subgraph.yaml

```
@@ -0,0 +1,43 @@
+specVersion: 0.0.2
+description: NFT Markets
+repository: https://github.com/pancakeswap
+schema:
+  file: ./schema.graphql
+dataSources:
+  - kind: ethereum/contract
+    name: CollectibleMarketV1
+    network: bsc
+    source:
+      address: '0x0000000000000000000000000000000000000000' # To be updated
+      abi: CollectibleMarketV1
+      startBlock: 10000000 # To be updated
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.4
+      language: wasm/assemblyscript
+      file: ./mappings/index.ts
+      entities:
+        - Collection
+        - NFT
+        - Transaction
+        - User
```

```suggestion
        - Collection
        - NFT
        - User
        - Transaction
```
You need to define a `User` before a `Transaction` due to relationships.